packages/ember/addon/instance-initializers/sentry-performance.ts

```diff
@@ -29,10 +29,13 @@ export function initialize(appInstance: ApplicationInstance): void {
 }
 
 function getBackburner() {
+  if (_backburner) {
+    return _backburner;
+  }
+
   if (run.backburner) {
     return run.backburner;
   }
-  return _backburner;
```

Comment:
With this change not all code paths here will return a value - so `getBackburner()` might return undefined now. Can we get around this somehow? Or do we have to do larger refactors for the usage of the `getBackburner` function?

This is why https://github.com/getsentry/sentry-javascript/runs/5235458452?check_suite_focus=true is failing.

Reply:
That's a good point. I am not knowledgable enough in the history of Ember, but I believe that if `_backburner` does not exist, it means we are on an older version of Ember and thus `run.backburner` exists. We could thus remove the condition around `run.backburner`.